rct_optimizations/test/pnp_utest.cpp

```diff
@@ -126,22 +142,14 @@ TEST(PNP_3D, PerfectInitialConditions)
   EXPECT_TRUE(result.camera_to_target.isApprox(target_to_camera.inverse()));
   EXPECT_LT(result.initial_cost_per_obs, 1.0e-15);
   EXPECT_LT(result.final_cost_per_obs, 1.0e-15);
+  checkCorrelation(result.camera_to_target_covariance);
+
+//  Eigen::IOFormat fmt(4, 0, " | ", "\n", "|", "|");
+//  std::cout << "Covariance:\n" << result.camera_to_target_covariance.format(fmt) << std::endl;
```

Comment:
Do you want to leave this commented out? Might be worthwhile to print during the tests anyway.